cypress/integration/navigation.spec.js

```diff
@@ -0,0 +1,9 @@
+const URL = 'localhost:3000';
+context('Navigation', () => {
+  beforeEach(() => {
+    cy.visit(URL);
+  });
+  it('.children() - get child DOM elements', () => {
+    cy.get('nav').children('ul').should('not.have.length', 0);
```

Comment:
I think adding a data attribute element to the nav and targeting it with that could be more reliable for targeting the element. 
`data-cy="nav-list"` and  `[data-cy="nav-list]` for targeting the element. 
more information can be found here https://docs.cypress.io/guides/references/best-practices.html

Reply:
This is so cool, I didn't know you could do this! I'm thinking of putting the <code>data-cy="nav-list"</code> attribute in the Box component from the SideNav.js file, since it has an attribute of <code>as='nav'</code>. Does that seem like the right place, @SiTaggart

Reply:
Yup

Reply:
It does not like when I do this but that attribute in the Box component from the SideNav.js file. When the code is:
<img width="292" alt="Screen Shot 2021-02-03 at 2 48 07 PM" src="https://user-images.githubusercontent.com/77117846/106819569-d6034780-662e-11eb-8e00-3b4e37977189.png">

I get this error:
![Screen Shot 2021-02-03 at 2 46 55 PM](https://user-images.githubusercontent.com/77117846/106819450-ace2b700-662e-11eb-926a-3bc41235435e.png)